test/runTest.ts

```diff
@@ -1,48 +1,57 @@
-import * as cp from "child_process";
 import * as path from "path";
 import {
-    downloadAndUnzipVSCode,
-    resolveCliArgsFromVSCodeExecutablePath,
     runTests,
+    runVSCodeCommand,
 } from "@vscode/test-electron";
 
 async function main() {
     try {
+        const vscodeVersion = "insiders";
         const extensionDevelopmentPath = path.resolve(__dirname, "../../");
         const extensionTestsPath = path.resolve(__dirname, "index");
 
-        // Add the dependent extension for test coverage preview functionality
-        const vscodeExecutablePath = await downloadAndUnzipVSCode("insiders");
-        const [cliPath, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath);
+        // Ensure live-server is not installed
```

Comment:
I'm not much of a fan of this as it means that we end up running all the tests twice, so this roughly doubles the amount of time it takes to run the tests, but wasn't able to find a better way of running a test without the live server extension installed...

Reply:
If you wrap `const livePreview = extensions.getExtension("ms-vscode.live-server");` in a function, you can stub that function in a couple of tests and get rid of all of this.

Reply:
or leave it installed and only stub it being missing in one test